### PR TITLE
Update add_customer_match_user_list example

### PR DESCRIPTION
**Pull request by BenRKarl**
Adds a demonstration of how to get the lists upload status without polling.

Fixes: #378

### CODE REVIEW

**Review by wihl — APPROVED** [review]
s/google_ads_/googleads_/ throughout, otherwise LGTM

File: examples/remarketing/add_customer_match_user_list.py [hunk 1]
```diff
@@ -249,6 +242,66 @@ def _build_offline_user_data_job_operations(client):
...
+    if status_name == "SUCCESS":
+        _print_customer_match_user_list_info(
+            client, customer_id, user_list_resource_name
+        )
+    elif status_name == "FAILED":
+        print(f"\tFailure Reason: {offline_user_data_job.failure_reason}")
+    elif status_name in ("PENDING", "RUNNING"):
+        print(
+            "To check the status of the job periodically, use the following "
+            f"GAQL query with GoogleAdsService.search: {query}"
```
Comment (laurachevalier4) [inline examples/remarketing/add_customer_match_user_list.py:301]: Capitalize `Search`

Comment (laurachevalier4) [reply examples/remarketing/add_customer_match_user_list.py:301]: Or maybe make it google_ads_service.search for consistency with above. These are nits though

**Review by laurachevalier4 — APPROVED** [review]
nit comment, otherwise lgtm

**Pull request merged**